src/main.js

```diff
@@ -45,11 +45,8 @@ Apify.main(async () => {
         proxyConfiguration,
         launchContext: {
             launchOptions: {
-                waitUntil: 'load',
-                // headless: true,
+                headless: true,
```

Comment:
Don't force headless mode. It is nice to have it off locally. By default it is on on platform and off locally